Genieo.adf

```diff
@@ -51,6 +51,9 @@
 		<File>/Applications/Totiteck.app</File>
 		<File>/Applications/Uninstall Genieo.app</File>
 		<File>/Applications/Uninstall IM Completer.app</File>
+		<File>/Applications/Epolife</File>
+		<File>/Applications/FriendlySocket</File>
```

Comment:
I'm pretty sure savingthrow is going to need the actual file name. Was it `Epolife` or `Epolife.app`?